docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -176,22 +181,27 @@ to be rude to one’s fellow players.
 
 When the GM or a Player calls for a Check, they announce who will be making this check
 and reveal (a) a Target Card (TC), (b) the Attribute or Skill associated with the
-check, and (c) the Difficulty Range (DR).
+check. They may also choose to reveal (c) the Difficulty Range (DR).
 
 A Target Card (TC) is like the bullseye on a target you are hoping to hit. The
 Difficulty Range (DR) is the range around the TC the player might hope to draw, like
 the circles around the bullseye.
 
-Once the DR has been announced, players must make the check. Players may consult with
-their GM to determine if another Attribute or Skill may be used instead. A player will
-draw one or more cards. Immediately after drawing, a player will apply any relevant
-Modifiers before reporting the outcome and whether or not they have successfully fallen
-within the DR. A Modifier is added to the DR when a check is made. A Modifier is
-indicated on the character sheet next to the relevant Skill or Attribute related to the
-Check. Think of it as a character's ability to 'aim' a check to get closer to the TC.
-
-The GM may describe the nature of the outcome or announce the degree of success and ask
-the player to describe how their character successfully accomplishes the action.
+The Difficulty Range is decided by the GM and they can choose whether or not to reveal
+it before a check is made. Players may consult with their GM to determine if another
+Attribute or Skill may be used instead. Once the check is called for and the relevant
+skill is selected, players must make the check. A player will draw one or more cards.
+Immediately after drawing, a player will apply any relevant Modifiers before reporting
+the outcome and whether or not they have successfully fallen within the DR. A Modifier
+is added to the DR when a check is made, effectively widening the DR and making it
+easier to succeed. A Modifier is indicated on the character sheet next to the relevant
+Skill or Attribute related to the Check. Think of it as a character's ability to 'aim' a
+check to get closer to the TC.
+
+The GM may describe the nature of the outcome or announce the degree of success. This
+may be simply to report the success or failure of an action, or provide a more detailed
+description of the outcome. The GM may ask the player to describe how their character
+successfully accomplishes the action depending on the scenario.
```

Comment:
I had edited out the `this is the character's ability to 'aim' a check to get closer to the TC` since we decided that the Mod should widen the DR and make the Effective DR a higher number. So it's less about Aiming and more about widening the DR to make it easier to succeed. I'll revisit this code suggestion to see how we resolve both, since I see widening the DR is included towards the bottom of the first pgf.